ui/src/message_popup/author/new_question_page.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import QuestionPage from './question_page.jsx';
+import {allArchivedQuestions} from './archived_questions.js';
+import {allQuestions} from '../questions.js';
+import * as Api from '../../helpers/api.js';
+
+
+export default React.createClass({
+  displayName: 'NewQuestionPage',
+
+  getInitialState(){
+    return ({
+      loaded: false,
+      allDatabaseQuestions: {
+        currentQuestions: allQuestions,
+        archivedQuestions: allArchivedQuestions
+      }
+    });
+  },
+
+  componentDidMount(){
+    Api.questionsQuery().end(this.onQuestionsReceived);
+  },
+
+  onQuestionsReceived(err, response){
+    const questions = JSON.parse(response.text).questions;
+    if(questions !== undefined){
```

Comment:
Yeah having the questions in `getInitialState` makes sense for initial UI testing, but that won't be like that going forward.

For err, the `API` methods are ultimately making HTTP requests using a library called superagent, and it follows the node style for passing errors back, which you can see more about here: https://visionmedia.github.io/superagent/#error-handling

I think the root issue may be that we should update the endpoint you added in https://github.com/mit-teaching-systems-lab/threeflows/pull/114 so that it returns `{ questions: [] }` to match the changes in this PR.